test-cases/tools/cmd/lint.ts

```diff
@@ -26,6 +26,9 @@ type error = string | null;
 
 const AUTOMATION = /^[A-Z]+-[0-9]+$/;
 
+// Update the README.md too
```

Comment:
Have you updated the README.md? :)

Reply:
who reads the documentation? :P

will fix it in next commit